<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="cpuCheckBox13">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 13</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartProcessClicked">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Process</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetPriority">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Priority</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SuspendProcess">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Suspend Process</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Create new file</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Set only read permission</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Set write permission</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Set SID Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Set SID Administrator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Set SID Group</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 11</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 7</string>
    </property>
   </widget>
   <widget class="QComboBox" name="processComboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ResumeProcess">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TerminateProcess">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminate Process</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetAffinity">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Affinity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCheckBox1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU 1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
